PAWS/Contacts/Contacts.swift

```diff
@@ -13,70 +13,80 @@ import SwiftUI
-                givenName: "Aydin",
-                familyName: "Zahedivash"
+                    )
+                ]
             ),
-            image: Image("AydinSquarePhoto"), // swiftlint:disable:this accessibility_label_for_image
-            title: "Pediatric Stanford Cardiology Fellow",
-            description: String(localized: "AYDIN_ZAHEDIVASH_BIO"),
-            organization: "Stanford University",
-            address: {
-                let address = CNMutablePostalAddress()
-                address.country = "USA"
-                address.state = "CA"
-                address.postalCode = "94304"
-                address.city = "Palo Alto"
-                address.street = "725 Welch Rd"
-                return address
-            }(),
-            contactOptions: [
-                .email(addresses: ["aydinz@stanford.edu"]),
-                ContactOption(
-                    image: Image(systemName: "safari.fill"), // swiftlint:disable:this accessibility_label_for_image
-                    title: "Website",
-                    action: {
-                        if let url = URL(string: "https://profiles.stanford.edu/aydin-zahedivash?tab=bio") {
-                           UIApplication.shared.open(url)
+            Contact(
+                name: PersonNameComponents(
+                    givenName: "Aydin",
+                    familyName: "Zahedivash"
+                ),
+                image: Image("AydinSquarePhoto"), // swiftlint:disable:this accessibility_label_for_image
+                title: "Pediatric Stanford Cardiology Fellow",
+                description: String(localized: "AYDIN_ZAHEDIVASH_BIO"),
+                organization: "Stanford University",
+                address: {
+                    let address = CNMutablePostalAddress()
+                    address.country = "USA"
+                    address.state = "CA"
+                    address.postalCode = "94304"
+                    address.city = "Palo Alto"
+                    address.street = "725 Welch Rd"
+                    return address
+                }(),
+                contactOptions: [
+                    .email(addresses: ["aydinz@stanford.edu"]),
+                    ContactOption(
+                        image: Image(systemName: "safari.fill"), // swiftlint:disable:this accessibility_label_for_image
+                        title: "Website",
+                        action: {
+                            if let url = URL(string: "https://profiles.stanford.edu/aydin-zahedivash?tab=bio") {
+                                UIApplication.shared.open(url)
+                            }
                         }
-                    }
-                )
-            ]
-        )
-    ]
+                    )
+                ]
+            )
+        ]
+        
+        if enrollmentGroup.studyType == .adult {
+            result.append(Contact(name: PersonNameComponents(givenName: "Brynne")))
+        }
```

Comment:
I think that the contracts will be different between the studies; we need to check this with Aydin & Scott. I would suggest to make this a switch statement that switches on the type of study 👍